config/locales/en.yml

```diff
@@ -110,6 +110,8 @@ en:
       date_order: "Date of birth must be in the past"
       email_format: "Enter email address in the correct format, like name@example.com"
       postcode_format: "Enter a real postcode"
+      negative_date: "Enter a real %{field}"
+      date_not_a_number: "Enter %{field} as a number"
```

Comment:
@m-green any opinions on this?

Example: "Enter day as a number"

Reply:
I think these came from some previous content designer's work on another form - so I think if they're happy, I'm happy.

Reply:
This was all copied from the Brexit funding form.  I believe the logic was we couldn't use "Enter a real month", because someone entering "June" is still entering a real month albeit not the correct format.